src/stream/src/executor/monitor/streaming_stats.rs

```diff
@@ -234,6 +235,14 @@ impl StreamingMetrics {
         )
         .unwrap();
 
+        let source_backfill_row_count = register_int_counter_vec_with_registry!(
+            "stream_source_backfill_rows_counts",
+            "Total number of rows that have been backfilled for source",
+            &["source_id", "source_name", "actor_id", "fragment_id"],
```

Comment:
This is just like the one in BackfillExecutor.

> the consumption progress for each partition.

This metrics is included in reader. So actually it also applies to Backfill. But it may be a little confusing, cause some metrics contain both Backfill and Source but others don't. I will show you an example later.

Reply:
This is what it looks like. Some metrics are inside the source reader, so it applies to both. Others are inside executor.

<img width="1633" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/37948597/57d66981-f36e-460e-9ed2-c13d8f8629b9">